qemu/tests/qemu_guest_agent_snapshot.py

```diff
@@ -41,18 +43,38 @@ def _action_before_fsfreeze(self, *args):
             dict(verbose=True, timeout=copy_timeout))
         self.bg.start()
 
+    def check_snapshot(self):
+        """
+        Check whether the snapshot is created successfully.
+        """
+        snapshot_info = str(self.vm.monitor.info("block"))
+        snapshot_node_name = self.params.get("snapshot_node_name")
+        if self.params.get("snapshot_file") not in snapshot_info:
+            logging.error(snapshot_info)
+            raise exceptions.TestFail("Snapshot doesn't exist")
```

Comment:
what about self.test.fail('Snapshot doesn't exist: %s' % snapshot_info), then we don't need to import exceptions, and one line can express the failure well

Reply:
+1

Reply:
and any update for this?

Reply:
updated to self.test.fail("Snapshot doesn't exist:%s" % snapshot_info)